cps/static/js/main.js

```diff
@@ -239,4 +239,17 @@ $(function() {
         $(".discover .row").isotope("layout");
     });
 
+    $(".update-view").click(function(e) {
+        var target = $(this).data('target');
+        var view = $(this).data('view');
+
+        e.preventDefault();
+        e.stopPropagation();
+        data = {};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['data' is not defined.](https://app.codacy.com/manual/Ozzie-IsaacsOrganization/calibre-web/pullRequest?prid=4846309)